include/picongpu/plugins/output/images/PngCreator.tpp

```diff
@@ -41,8 +41,10 @@
 
 namespace picongpu
 {
-    template<class Box>
-    inline void PngCreator::createImage(const Box data, const MessageHeader::Size2D size, const MessageHeader header)
+    template<typename T_DataType>
+    inline void PngCreator::createImage(
+        std::shared_ptr<std::vector<T_DataType>> imageVector,
```

Comment:
Shouldn't you add a `#include <vector>` to this file?

Reply:
added